corehq/apps/reports/standard/cases/filters.py

```diff
@@ -90,7 +90,7 @@ class XPathCaseSearchFilter(BaseSimpleFilter):
     """
     slug = 'search_xpath'
     label = gettext_lazy("Search")
-    template = "reports/filters/bootstrap3/xpath_textarea.html"
+    template = "reports/filters/bootstrap5/xpath_textarea.html"
```

Comment:
I'm pretty certain this is wrong, because this filter is used in non-Bootstrap 5 places (`CaseListExplorer`, and `DuplicateCasesExplorer`). Is there a way to resolve this?